src/components/Pricing/PricingExperiment.tsx

```diff
@@ -0,0 +1,1216 @@
+                                <div className="flex flex-col md:grid grid-cols-2 lg:flex justify-between h-full">
+                                    <PlanContent
+                                        activePlan={activePlan}
+                                        onFreeTierClick={() => setAnimateFreeTiers(true)}
+                                    />
+                                </div>
+                            </div>
+                        </aside>
+                    </div>
+
+                    <PaidPricing />
+
+                    <PlanColumns billingProducts={billingProducts} highlight="free" />
+
+                    <Addons />
+                    <SimilarProducts />
+                    <PurchasedWith />
+                    <Reviews />
+
+                    <SectionLayout>
+                        <SectionHeader>
+                            <h3 id="calculator">Pricing calculator</h3>
+                        </SectionHeader>
+                        <Tabbed />
+
+                        <div className="grid md:grid-cols-2 gap-8 mt-12 max-w-7xl mx-auto">
+                            <div>
+                                <h4 className="text-lg mb-2">How our pricing works</h4>
+                                <SidebarList>
+                                    <SidebarListItem>Only pay for products you use</SidebarListItem>
+                                    <SidebarListItem>
+                                        <strong className="bg-yellow/50 dark:bg-white/20 italic inline py-0.5">
+                                            Generous free tier for each product (resets monthly)
+                                        </strong>
+                                    </SidebarListItem>
+                                    <SidebarListItem>
+                                        You can set billing limits per product so you never get a surprise bill
+                                    </SidebarListItem>
+                                    <SidebarListItem>
+                                        We also offer{' '}
+                                        <Tooltip content={() => <Discounts />} placement="top">
+                                            <strong className="text-red dark:text-yellow border-b border-dashed border-light dark:border-dark cursor-help text-primary/75 dark:text-primary-dark/75">
+                                                discounts
+                                            </strong>
+                                        </Tooltip>{' '}
+                                        for startups and non-profits
+                                    </SidebarListItem>
+                                </SidebarList>
+                            </div>
+                            <div>
+                                <h4 className="text-lg mb-2">Estimating usage</h4>
```

Comment:
We should link this section to https://posthog.com/docs/billing/estimating-usage-costs

Sales has asked us to link it as well